manifests/d/DOSBoxStaging/DOSBoxStaging/0.80.1/DOSBoxStaging.DOSBoxStaging.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.3.4 $debug=NVS1.CRLF.5-1-19041-4123.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DOSBoxStaging.DOSBoxStaging
+PackageVersion: 0.80.1
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/dosbox-staging/dosbox-staging/releases/download/v0.80.1/dosbox-staging-v0.80.1-setup.exe
+  InstallerSha256: 94B961F397F3B70A13DAF3B573857B5262515E75D490BFC19F019479FE1694F8
+  UpgradeBehavior: install
+- Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: dosbox-staging-v0.81.0\dosbox.exe
+  InstallerUrl: https://github.com/dosbox-staging/dosbox-staging/releases/download/v0.81.0/dosbox-staging-windows-v0.81.0.zip
+  InstallerSha256: CE772A963716D63610E6CB0E817B16F1B5A3CBFD3D5AD802726DDD583F32B79D
```

Comment:
```suggestion
```

Reply:
Why including both the regular installer and portable types?

Reply:
Because winget allows you to include either the normal installation or a portable version.